packages/manager/modules/pci/src/projects/project/instances/instance/shelve/shelve.controller.js

```diff
@@ -0,0 +1,76 @@
+import get from 'lodash/get';
+
+export default class PciInstanceShelveController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    OvhApiCloudProjectInstance,
+    PciProjectsProjectInstanceService,
+    Poller,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OvhApiCloudProjectInstance = OvhApiCloudProjectInstance;
+    this.PciProjectsProjectInstanceService = PciProjectsProjectInstanceService;
+    this.Poller = Poller;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+  }
+
+  startPollerForShelveInstance() {
+    const endPointUrl = `/cloud/project/${this.projectId}/instance/${this.instance.id}`;
+    this.Poller.poll(endPointUrl, null, {
+      interval: 3000,
+      successRule(instance) {
+        return instance.status === 'SHELVED_OFFLOADED';
```

Comment:
Not possible to reuse this method: https://github.com/ovh/manager/pull/4401/files#diff-895fbc1eb2ccc052e43a524cfe672890f170da64759693f7a1abd0bb7f5966b9R210

Reply:
Yes we can.
I hesitate, because we create a new Instance for each new poller call.

Reply:
Fixed.